packages/playground/website/src/components/site-manager/site-manager-sidebar/style.module.css

```diff
@@ -0,0 +1,94 @@
+.site-manager-sidebar {
+	height: 100vh;
+	display: flex;
+	flex-direction: column;
+	--site-manager-text-color: #fff;
+	color: var(--site-manager-text-color);
+}
+
+.site-manager-sidebar-header {
+	display: flex;
+	padding: 16px 24px;
```

Comment:
I [Disabled logo clicks](https://github.com/WordPress/wordpress-playground/pull/1661/commits/3728dd22e40e76e4ddf3274e3754b564a37d4dba) because this is how the design was initially created.

Making them overlap would require redesigning the sidebar.

I'm also considering disabling the view again to make it work as planned in the design.

Reply:
I will discuss this with @jarekmorawski today.

Reply:
> I [Disabled logo clicks](https://github.com/WordPress/wordpress-playground/pull/1661/commits/3728dd22e40e76e4ddf3274e3754b564a37d4dba) because this is how the design was initially created.

This seems like a fine solution, but AFAICT, there is now no way to close the sidebar, even if a site is clicked. Should the sidebar collapse once a site (even the current site) is clicked?

Reply:
Hmm, this might be the same concern you mentioned [here](https://github.com/WordPress/wordpress-playground/pull/1661#issuecomment-2268536956).